core/templates/pages/library-page/search-bar/search-bar.component.spec.ts

```diff
@@ -200,6 +213,18 @@ describe('Search bar component', () => {
     fixture.detectChanges();
   });
 
+  it('should determine if mobile view is active', () => {
```

Comment:
we should also check for a `false` condition

Reply:
@aishwary023 I've modified the tests to now check for a false value. PTAL!